setup.cfg

```diff
@@ -1,16 +1,79 @@
+[metadata]
+name = nlp_primitives
+version = attr: nlp_primitives.version.__version__
+author = Alteryx Inc.
+license = BSD 3-clause
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/alteryx/nlp_primitives
+author_email = open_source_support@alteryx.com
+classifiers =
+    Intended Audience :: Science/Research
+    Intended Audience :: Developers
+    Programming Language :: Python
+    Topic :: Software Development
+    Topic :: Scientific/Engineering
+    Development Status :: 5 - Production/Stable
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX
+    Operating System :: Unix
+    Operating System :: MacOS
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+
+[options]
+packages = find:
+python_requires = >=3.7,<4
+include_package_data = True
+zip_safe = False
+
+install_requires =
+    numpy >= 1.17.5
+    pandas >= 1.3.0
+    featuretools >= 1.5.0
+    nltk >= 3.4.5
+    scikit-learn >= 0.20.0
+
+[options.extras_require]
+complete =
+    tensorflow >= 1.14.0; sys_platform!="darwin" or platform_machine!='arm64'
+    tensorflow-metal >= 0.4.0; sys_platform=="darwin" and platform_machine=='arm64'
+    tensorflow-macos >= 2.8.0; sys_platform=="darwin" and platform_machine=='arm64'
```

Comment:
would be useful to confirm if those constraints are being recognized

Reply:
I tested by running `pip install -e ".[complete]" on my M1 mac and got these in my env, so seems to be working.

```
tensorflow-macos==2.8.0
tensorflow-metal==0.4.0
```

Reply:
My non-M1 machine did not install those so seems to be working!

Reply:
@thehomebrewnerd I thought `tensorflow-metal` and `tensorflow-macos` are deprecated? Does tensorflow not support M1 Macs natively yet?

Reply:
As I understand it, there are no pre-built wheels of `tensorflow` that work on M1 macs, so you would have to build it yourself, but I think it would support M1 in that case.

https://github.com/tensorflow/tensorflow/issues/52973

On my M1:
```
>>> pip install tensorflow
ERROR: Could not find a version that satisfies the requirement tensorflow (from versions: none)
ERROR: No matching distribution found for tensorflow
```

Reply:
Gotcha, let's leave this